specification/cognitiveservices/OpenAI.Inference/models/completions/extensions_onyourdata_common_options.tsp

```diff
@@ -0,0 +1,30 @@
+import "@typespec/versioning";
+import "./azure_chat_extensions.tsp";
+import "./extensions_onyourdata_authentication.tsp";
+
+namespace Azure.OpenAI;
+
+using TypeSpec.Versioning;
+
+alias OnYourDataCommonChatExtensionConfiguration = {
+  @doc("The authentication option to access the data.")
+  @projectedName("csharp", "Authentication")
+  authentication?: OnYourDataAuthenticationOptions;
```

Comment:
Should this property be required? 🤔

Reply:
We use "system_assigned_managed_identity" as default authentication, if not provided explicitly. Then the request can be simpler and curl friendly. Good point - we should document it clearly what is the default authentication.

Reply:
Updating to this; wordy, but hopefully helpful:
```
  @doc("""
  The authentication method to use when accessing the defined data source.
  Each data source type supports a specific set of available authentication methods; please see the documentation of
  the data source for supported mechanisms.
  If not otherwise provided, On Your Data will attempt to use System Managed Identity (default credential)
  authentication.
  """)
```